src/types/session.types.ts

```diff
@@ -0,0 +1,17 @@
+export interface User {
```

Comment:
Naming this interface something like `DecodedToken` would probably be more accurate as the User probably doesn't need so many attributes. I think it's more appropriate to create two interfaces, one `DecodedToken` for defining JWT decoding, one `User` which defines only attributes strictly required for display.